File: contact/php/process_form.php
<?php
if ($_SERVER["REQUEST_METHOD"] == "POST") {
  $name = $_POST["name"];
  $company = $_POST["company"];
  $country = $_POST["country"];
  $subject = $_POST["subject"];

  $to = "cases@gmx.de";  // Replace with your email address
  $subject = "New Contact Form Submission";
  $message = "Name: $name\nCompany: $company\nCountry: $country\n\nSubject:\n$subject";

  // Use additional headers if needed
  $headers = "From: cases@gmx.de";  // Replace with your email or leave it empty

  // Send the email
  mail($to, $subject, $message, $headers);

  // Redirect to a thank you page or back to the main page
  header("Location: ../contact/index.html");  // Create thank_you.html with your thank you message
  exit();
}
?>

File: contact/py/process_form.py
from flask import Flask, request, redirect, render_template
from flask_mail import Mail, Message

app = Flask(__name__)

# Configure Flask-Mail
app.config['MAIL_SERVER'] = 'smtp.gmail.com'
app.config['MAIL_PORT'] = 587
app.config['MAIL_USE_TLS'] = True
app.config['MAIL_USERNAME'] = 'vali.pinocasso@gmail.com'
app.config['MAIL_PASSWORD'] = '13youtube22001'
app.config['MAIL_DEFAULT_SENDER'] = 'vali.pinocasso@gmail.com'
app.config['MAIL_DEBUG'] = True

mail = Mail(app)

@app.route('/submit_form', methods=['POST'])
def submit_form():
    if request.method == 'POST':
        name = request.form['name']
        company = request.form['company']
        country = request.form['country']
        subject = request.form['subject']

        to_email = "vali.pinocasso@gmail.com"
        email_subject = "New Contact Form Submission"
        email_message = f"Name: {name}\nCompany: {company}\nCountry: {country}\n\nSubject:\n{subject}"

        # Send the email using Flask-Mail
        message = Message(email_subject, recipients=[to_email])
        message.body = email_message
        mail.send(message)

        # Redirect to a thank you page or back to the main page
        return redirect('../contact/index.html')

if __name__ == '__main__':
    app.run(debug=True)
